projects/js-packages/number-formatters/package.json

```diff
@@ -0,0 +1,43 @@
+{
+	"name": "@automattic/number-formatters",
+	"version": "0.1.0",
```

Comment:
The "changelogger validity" warning from CI here should be fixed by the suggestion I made for CHANGELOG.md.